setup.py

```diff
@@ -48,7 +48,7 @@ def get_packages(package):
     packages=get_packages("databases"),
     package_data={"databases": ["py.typed"]},
     data_files=[("", ["LICENSE.md"])],
-    install_requires=['sqlalchemy<1.4', 'aiocontextvars;python_version<"3.7"'],
+    install_requires=['sqlalchemy>=1.4,<1.5', 'aiocontextvars;python_version<"3.7"'],
```

Comment:
As far as I understand your change is backwards compatible with older versions of `sqlalchemy`. Do we need to pin this to `>= 1.4`?

Reply:
No it's not. It will break for example when importing `sqlalchemy.engine.cursor` (see the related issue)
I asked above if I should make it compatible (in which case I would have done a util module that imports the right methods depending on the version) but [we agreed](https://github.com/encode/databases/pull/299#issuecomment-832555219) it was not necessary. If people want to use sqlachemy 1.3, pip will install `databases 0.4.3`

Reply:
Sorry, I missed that.